src/utils/selectedCellUtils.ts

```diff
@@ -60,6 +62,13 @@ export function getSelectedCellColSpan<R, SR>({
     return getColSpan(column, lastFrozenColumnIndex, { type: 'HEADER' });
   }
 
+  if (topSummaryRows) {
+    return getColSpan(column, lastFrozenColumnIndex, {
```

Comment:
Shouldn't we only return if this returned a value?

Reply:
It was a bug 👍